custom/icds/translations/integrations/transifex.py

```diff
@@ -70,19 +72,38 @@ def client(self):
         else:
             raise Exception(_("Transifex account details not available on this environment."))
 
+    @property
+    @memoized
+    def transifex_project_source_lang(self):
+        return self.client.transifex_lang_code(self.client.get_source_lang())
+
+    def _resource_name_in_project_lang(self, resource_slug):
+        """
+        return the name of the resource i.e module/form in source lang on Transifex
+
+        :param resource_slug: like module_moduleUniqueID
+        """
+        if MODULES_AND_FORMS_SHEET_NAME in resource_slug:
+            return MODULES_AND_FORMS_SHEET_NAME
+        module_or_form_unique_id = resource_slug.split('_')[1]
+        resource_name_in_all_langs = self.transifex_po_file_generator.slug_to_name[module_or_form_unique_id]
+        return resource_name_in_all_langs.get(self.transifex_project_source_lang,
+                                              resource_name_in_all_langs.get('en', resource_slug))
```

Comment:
Curious why `'en'` is used here? Is there a default language for the app or domain? Would it make sense to use that instead?

Reply:
Nothing in particular. Just used english as default, though its highly unlikely this would ever happen since the transifex project source lang should always be same as default language on HQ app for transifex integration to work, which in the app, should always be present. 
So just used "en" in case someone tries something out of the box.